src/Error/ErrorBuilder.php

```diff
@@ -33,16 +33,21 @@ final class ErrorBuilder
 
     /**
      * @param string[] $errorPatternsToIgnore
+     * @param array<string, string> $applicationMapping
      * @param ErrorTransformerInterface[] $errorTransformers
      */
     public function __construct(
         array $errorPatternsToIgnore,
+        array $applicationMapping,
         array $errorTransformers,
         LineMapper $lineMapper
     ) {
         $this->errorPatternsToIgnore = array_merge($this->errorPatternsToIgnore, $errorPatternsToIgnore);
         $this->errorTransformers = $errorTransformers;
         $this->lineMapper = $lineMapper;
+        if (count($applicationMapping) === 0) {
+            $this->errorPatternsToIgnore[] = '/Cannot load presenter .*/';
```

Comment:
toto mozeme ignorovat vzdy nie?

Reply:
To by prave ignorovalo tu chybu co vyhazuju kdyz je chybny link